ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/operations/ProposerSlashingTest.java

```diff
@@ -56,7 +57,7 @@ void equalsReturnsTrueWhenObjectFieldsAreEqual() {
   void equalsReturnsFalseWhenProposerIndicesAreDifferent() {
     ProposerSlashing testProposerSlashing =
         new ProposerSlashing(
-            proposerIndex + randomInt(),
+            randomUnsignedLong(),
```

Comment:
I don't like testing a random value against another random value, since there's a chance they're the same, invalidating the test.
```suggestion
            proposerIndex.plus(randomUnsignedLong()),
```

Reply:
Makes sense, thanks.